packages/plugin-survey/src/index.ts

```diff
@@ -529,271 +182,24 @@ class SurveyPlugin implements JsPsychPlugin<Info> {
         }
       }
 
-      // TO DO: restructure survey data (sender.data) here?
+      // clear display and reset flex on jspsych-content-wrapper
+      display_element.innerHTML = "";
+      $(".jspsych-content-wrapper").css("display", "flex");
+
       // finish trial and save data
       this.jsPsych.finishTrial({
-        rt: Math.round(performance.now() - start_time),
+        rt: Math.round(performance.now() - this.start_time),
         response: sender.data,
-        accuracy: this.trial_data.accuracy,
       });
     });
-  }
 
-  /**
-   * Validate parameters for any question type
-   *
-   * @param supplied
-   * @param required
-   * @param optional
-   * @returns
-   */
-  private static validate_question_params(
-    supplied: Record<string, unknown>,
-    required: string[],
-    optional: string[]
-  ) {
-    required = [...all_question_params_req, ...required];
-    optional = [...all_question_params_opt, ...optional];
-
-    for (const param of required) {
-      if (!supplied.hasOwnProperty(param)) {
-        throw new Error(
-          param === "type"
-            ? 'Error in survey plugin: question is missing the required "type" parameter.'
-            : `Error in survey plugin: question is missing required parameter "${param}" for question type "${supplied.type}".`
-        );
-      }
-    }
+    // remove flex display from jspsych-content-wrapper to get formatting to work
+    $(".jspsych-content-wrapper").css("display", "block");
 
-    const invalid_params = Object.keys(supplied).filter(
-      (param) => !(optional.includes(param) || required.includes(param))
-    );
+    $(display_element).Survey({ model: this.survey });
```

Comment:
more jQuery to think about

Reply:
This bit of jquery would've been necessary with the survey-jquery package, but Bjorn convinced me to switch to survey-knockout-ui, so now it's gone. And I didn't have to replace it with any knockout-specific syntax or conventions. And I think the package size is smaller too. Yay!